### ISSUE TITLE
[TASK] Fix Lighthouse failures - address-input

### ISSUE THREAD

**Issue by MagdalenaLarge** [ONSdigital/design-team#215]
Alessio addressed Lighthouse failure in the ticket [#3350](https://github.com/ONSdigital/design-system/issues/3350).

As an outcome of the above ticket, identified a need to address the warnings for address input component. It relates to accessibility.

1 result(s) for http://localhost:39203/components/address-input/example-address-input.html :
1 result(s) for http://localhost:39203/patterns/addresses/example-address-input-non-editable.html :

**Action**
Use yarn start and then move onto the above components in order. Use the lighthouse test from the google chrome dev tools. Run the test on the component and only look at the accessibility score. Fix the issues highlighted under the accessibility section. Run the test again and confirm that they are now at 100. Move on to the next component.
